tests/test-team/template-mgmt-component-tests/template-mgmt-choose-page.component.ts

```diff
@@ -33,19 +34,29 @@ export const nhsAppRadioSelectedSessionData: Session = {
   nhsAppTemplateMessage: '',
 };
 
+const testUserEmail = 'choose-page@nhs.net';
+const testUserPassword = 'Test-Password1';
```

Comment:
No hard coded passwords please (even for test).

Reply:
updated